testing/trino-benchto-benchmarks/src/test/java/io/trino/sql/planner/AbstractCostBasedPlanTest.java

```diff
@@ -128,9 +128,9 @@ private static Path getSourcePath()
         verify(isDirectory(workingDir), "Working directory is not a directory");
         String topDirectoryName = workingDir.getFileName().toString();
         switch (topDirectoryName) {
-            case "trino-benchto-benchmarks":
+            case "presto-benchto-benchmarks":
```

Comment:
sure not "trino"?

Reply:
Thanks for catching. I changed more than was needed. Seems like we missed changing the `case "presto"` during rename.